### PR TITLE
Added hover effect to buttons

### LINKED ISSUE #2: Add hover effects to Header buttons - Easy Issue
Add hover effect whenever you hover the pause/play and mute button

### PR DESCRIPTION
**Pull request by BingBlingBoy**
Added simple grow and shrink hover effect to buttons

### CODE REVIEW

**Comment by Tsu-HaoLiu** [comment]
Thanks for the PR, this looks great but I think there are a few things missing. When hovering over the grey play button it doesn't seem to change to white. The mute button also isn't grey, its the same white. Can you double check this for me?

nvUSFuumgH

**Comment by BingBlingBoy** [comment]
> Thanks for the PR, this looks great but I think there are a few things missing. When hovering over the grey play button it doesn't seem to change to white. The mute button also isn't grey, its the same white. Can you double check this for me?
> 
> nvUSFuumgH

I've checked the problem and it is due to the svg's. I have limited knowledge of svg's and tailwind in general so I don't know the potential features that will change the color of the button when the user hovers over them (I'll research over the next upcoming days). I also have another idea where we can treat the svg's as components where we can change the fill property of them so for example `<svg fill={isClicked ? #ffffff : #c6c6c6} ....>`. Feel free to suggest any better solutions and I'll try and implement them to the best of my ability.

**Comment by Tsu-HaoLiu** [comment]
Hey @BingBlingBoy 👋, just doing a quick check in to see how its going with the research and to see if you need any help.

**Comment by BingBlingBoy** [comment]
> Hey @BingBlingBoy 👋, just doing a quick check in to see how its going with the research and to see if you need any help.

Sorry I took so long to reply. I got carried away learning tailwind and doing some frontend mentor challenges. 

Testing out the svg's as components idea. I managed to get the component to render but not as expected, currently looks like this:
Screenshot 2023-08-22 170021
The oversized mute button on the right is the component. I'll find out what is causing the component to be big as well as the blur in the background. Feel free to suggest any other ideas.

**Comment by BingBlingBoy** [comment]
I've got a working implementation now and will push my changes. I've turned all the svg's into react components but I've still kept the files with svg's so if they are completely unneeded they can be deleted. I also have a question as well:

- How does the mute button work? Is it supposed to completely stop any audio once pressed and only plays audio once you press it again? Because after you press the button and it turns gray, if you then move one of the sliders, the button is still gray when the audio is playing. 
- Similar issue with the pause and play buttons.

In short how do you want the buttons to handle these issues?
